src/ixdat/techniques/ms.py

```diff
@@ -949,12 +1114,9 @@ def gas_flux_calibration_curve(
         mol,
         mass,
         tspan_list=None,
-        selector_list=None,
```

Comment:
Why are you removing this nice functionality of choosing EC selector periods rather than having to always pass a tspan? I would miss it dearly!

Reply:
Ah is it because it should be in EC-MS only?

Reply:
I'm also a bit confused as to why you have two definitions of the method `gas_flux_calibration_curve` in the same file?

Reply:
For the first question: Yes! An `MSMeasurement` in general doesn't come with a selector...
For the second question: It's not elegant, but the first one (method of `MSMeasurement`) uses `spectro_inlets_quantification` and this one (method of `MSInlet`) uses ixdat's native MS calibration.

Reply:
I started to try refactoring in a way that moving the shared code to another function, but I haven't come up with a way of doing it that makes the module shorter, since there are so many arguments that need to get repeated anyway... happy if you give it a try! Will leave this comment unresolved.

Reply:
I just remembered, `MSInlet` is deprecated. I don't think it makes sense to have in ixdat, as the only inlet we have the capillary equation for is an SI chip, and that is covered in `spectro_inlets_quantification`. So next release won't have the repeated code :)